application/app/views/download_status/_actions.html.erb

```diff
@@ -1,9 +1,10 @@
-<div class="text-end sticky-top bg-white z-index-999">
-  <%= link_to(files_app_url(Configuration.download_root), class: 'btn btn-outline-dark btn-sm', target: '_blank') do %>
-    <i class="bi bi-folder-fill"></i><span class="ps-1"><%= t('.downloads_directory') %></span>
+<div class="text-end sticky-top bg-white z-index-999" aria-label="<%= t('.downloads_actions_bar') %>">
```

Comment:
similar to this one, something like: `.actions_bar.title.a11y.label`

Reply:
done